tests/API/XCCDF/unittests/test_fix_script_header.sh

```diff
@@ -22,15 +21,15 @@ script=$(mktemp -t ${name}.sh.XXXXXX)
 playbook=$(mktemp -t ${name}.yml.XXXXXX)
 
 profile_header1a="# Bash remediation role for profile $profile"
-profile_header1b="# Ansible remediation role for profile $profile"
+profile_header1b="# Ansible playbook for profile $profile"
```

Comment:
```suggestion
profile_header1b="# Ansible Playbook for $profile"
```

Reply:
- Playbook vs playbook
- Most ansible users won't know these originated from an SSG profile, so suggest dropping the term ``profile``

Reply:
I think they will easily identify that it comes from SSG, because after this there isn't a profile title, but there is the long XCCDF profile ID. The title is on next line now. In current OpenSCAP it looks like this:
```
###############################################################################
#
# Ansible remediation role for profile xccdf_org.ssgproject.content_profile_ospp
# Profile Title:  OSPP - Protection Profile for General Purpose Operating Systems
# Profile Description:
# This profile reflects mandatory configuration controls identified in the
# NIAP Configuration Annex to the Protection Profile for General Purpose
# Operating Systems (Protection Profile Version 4.2).
# 
# As Fedora OS is moving target, this profile does not guarantee to provide
# security levels required from US National Security Systems. Main goal of
# the profile is to provide Fedora developers with hardened environment
# similar to the one mandated by US National Security Systems.
#
# Benchmark ID:  xccdf_org.ssgproject.content_benchmark_FEDORA
# Benchmark Version:  0.1.41
#
# XCCDF Version:  1.2
#
# This file was generated by OpenSCAP 1.3.0 using:
# 	$ oscap xccdf generate fix --profile xccdf_org.ssgproject.content_profile_ospp --template urn:xccdf:fix:script:ansible xccdf-file.xml 
#
# This script is generated from an OpenSCAP profile without preliminary evaluation.
# It attempts to fix every selected rule, even if the system is already compliant.
#
# How to apply this remediation role:
# $ ansible-playbook -i "localhost," -c local playbook.yml
# $ ansible-playbook -i "192.168.1.155," playbook.yml
# $ ansible-playbook -i inventory.ini playbook.yml
#
###############################################################################
```

I think we could put the title to the first line and move the ID somewhere below. I imagine something like this:
```
###############################################################################
#
# Ansible Playbook for OSPP - Protection Profile for General Purpose Operating Systems
#
# Profile Description:
# This profile reflects mandatory configuration controls identified in the
# NIAP Configuration Annex to the Protection Profile for General Purpose
# Operating Systems (Protection Profile Version 4.2).
# 
# As Fedora OS is moving target, this profile does not guarantee to provide
# security levels required from US National Security Systems. Main goal of
# the profile is to provide Fedora developers with hardened environment
# similar to the one mandated by US National Security Systems.
#
# Profile ID: xccdf_org.ssgproject.content_profile_ospp
# Benchmark ID:  xccdf_org.ssgproject.content_benchmark_FEDORA
# Benchmark Version:  0.1.41
# XCCDF Version:  1.2
#
# This file was generated by OpenSCAP 1.3.0 using:
# 	$ oscap xccdf generate fix --profile xccdf_org.ssgproject.content_profile_ospp --template urn:xccdf:fix:script:ansible xccdf-file.xml 
#
# This Ansible Playbook is generated from an OpenSCAP profile without preliminary evaluation.
# It attempts to fix every selected rule, even if the system is already compliant.
#
# How to apply this Ansible Playbook:
# $ ansible-playbook -i "localhost," -c local playbook.yml
# $ ansible-playbook -i "192.168.1.155," playbook.yml
# $ ansible-playbook -i inventory.ini playbook.yml
#
###############################################################################
```